test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -206,7 +206,7 @@ Describe "New-Item with links" -Tags @('CI', 'RequireAdminOnWindows') {
         Remove-Item $FullyQualifiedLink -Force
     }
 
-    It "Should error correctly when failing to create a symbolic link" -Skip:($IsWindows -or $IsElevated) {
+    It "Should error correctly when failing to create a symbolic link" -Skip:($IsWindows) {
```

Comment:
Seems `New-Item -ItemType SymbolicLink` should works on Windows with elevated permissions - why we skip the test on Windows?

Reply:
I am not sure why the test was skipped for Windows. Any insights on that? Should it be run on Windows without elevation?

Reply:
Tests below show that we can create symbolic links on Windows with elevated permissions. So I believe we shouldn't skip the test on Windows without elevated permissions.

Reply:
Ilya is saying that `New-item -ItemType SymbolicLink` works on Windows, but requires elevation.  We should update this test to also work on Windows but requires elevation.